.github/workflows/deploy-function-app.yaml

```
@@ -0,0 +1,110 @@
+          popd
+
+      - name: 🔑 Azure CLI Login
+        uses: azure/login@7ddb5af1ef8758cf1353cf3b42f940aee27ba21c # v3.0.2
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+
+      - name: 🌐 Get GitHub Runner IP
+        id: get_ip
+        run: echo runner_ip=$(curl -s ipinfo.io/ip) >> $GITHUB_OUTPUT
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:16: Quote this to prevent word splitting

(shellcheck)